global-script.lua

```diff
@@ -1859,20 +1859,39 @@ function timePassesCo()
     timePassing = false
     return 1
 end
+function getReminderColor(name)
+    return string.match(name, "^(%a*)'s %a* Reminder Tokens")
+end
 function handlePiece(object, depth)
-    if string.sub(object.getName(),1,4) == "City" then
+    local isoOffset = Vector(-11.2, 2, -1)
+    local defOffset = Vector(-9.5, 2, -1)
+    local name = object.getName()
+    local spos = getObjectFromGUID(elementScanZones["Red"]).getPosition()
+    if string.sub(name,1,4) == "City" then
         object = resetPiece(object, Vector(0,180,0), depth)
-    elseif string.sub(object.getName(),1,4) == "Town" then
+    elseif string.sub(name,1,4) == "Town" then
         object = resetPiece(object, Vector(0,180,0), depth)
-    elseif string.sub(object.getName(),1,8) == "Explorer" then
+    elseif string.sub(name,1,8) == "Explorer" then
         object = resetPiece(object, Vector(0,180,0), depth)
-    elseif string.sub(object.getName(),1,5) == "Dahan" then
+    elseif string.sub(name,1,5) == "Dahan" then
         object = resetPiece(object, Vector(0,0,0), depth)
     elseif object.getName() == "Blight" then
         object = resetPiece(object, Vector(0,180,0), depth)
-    elseif string.sub(object.getName(),-7) == "Defence" then
+    elseif string.sub(name,-7) == "Defence" then
         if object.getLock() == false then object.destruct() end
         object = nil
+    elseif string.sub(name,-23) == "Isolate Reminder Tokens" then
+        if object.getLock() == false then
+             local color = getReminderColor(name)
+             spos = getObjectFromGUID(elementScanZones[color]).getPosition()
+             object.setPositionSmooth(Vector(spos.x,0,spos.z) + isoOffset)
```

Comment:
this effectively will start stacking the tokens right?

Reply:
Yes.